esmvalcore/_task.py

```diff
@@ -549,15 +553,12 @@ def _collect_provenance(self):
             'exit_on_ncl_warning',
             'input_files',
             'log_level',
-            'output_file_type',
```

Comment:
Same here.

Reply:
That would ignore recording their value in the provenance, so it would break the idea that all diagnostic script settings are recorded in the provenance